File: C#-Basics/07-Exercises/Exercise-01.cs
partial class Program
{
    static void Exercise01()
    {
        List<int> numeros = new List<int> { 10, 15, 22, 7, 8, 31, 40, 13 };
        int pares = 0;
        int impares = 0;

        WriteLine("Numeros de lista:");
        foreach (int n in numeros)
        {
            if (n % 2 == 0)
            {
                pares++;
            }
            else
            {
                impares++;
            }
        }
        WriteLine($"Numeros pares: {pares}");
        WriteLine($"Numeros impares: {impares}");
    }

}

File: JS-Basics/09-Exercises/Exercises/Exercise-01.js
const numeros = [15, 8, 42, 23, 4, 16];

let mayor = numeros[0];

for (let i = 0; i < numeros.length; i++) {
    if (numeros[i] > mayor) {
        mayor = numeros[i];
    }

}

console.log(mayor)